.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: hhvm
```

Comment:
I would say HHVM is not a top priority for now, but it would be nice to support it eventually.